### DISCUSSION TITLE
Python 3.7.9: No module named 'discord'

### DISCUSSION THREAD

**Discussion by joshikida** [Rapptz/discord.py#6145] [Q&A]
## Summary

I just switched from a web-based IDE(repl.it) to VS Code on my Mac running Catalina. I have installed 3.7.9 and set it as global, and of course installed discord.py with pip. However, running `import discord` still causes the following error to appear: `ModuleNotFoundError: No module named 'discord'`. 
I also have the following packages: https://i.stack.imgur.com/dlnTq.png

## Reproduction Steps

```import discord``` in VS Code running Python 3.7.9

## Expected Results

I expected discord to reflect its status as being installed.

## Actual Results

`ModuleNotFoundError: No module named 'discord'`
`Traceback (most recent call last):
  File "/Users/myname/Desktop/CM-Public/main.py", line 6, in <module>
    import discord
ModuleNotFoundError: No module named 'discord'`

## System Information

- Python v3.7.9-final
- discord.py v1.5.1-final
- aiohttp v3.6.3
- system info: Darwin 19.6.0 Darwin Kernel Version 19.6.0: Thu Jun 18 20:49:00 PDT 2020; root:xnu-6153.141.1~1/RELEASE_X86_64

**Comment by ghost** [comment]
Are you running the code directly from the VS Code terminal? If so, make sure you have selected the right interpreter, more info [here](https://code.visualstudio.com/docs/python/environments).
Try reloading VS Code with <kbd>Shift</kbd>+<kbd>Command</kbd>+<kbd>P</kbd> and select `Developer: Reload Window`.

However consider using a virtual environment. You can find more information on how to create one [here](https://docs.python.org/3/library/venv.html) but let me explain briefly how to do it.

Make sure you are on the root of your bot directory (I guess CM-Public) and run `python3 -m venv env`
Then you must activate it (everytime you need to use it) by running `source env/bin/activate`.
To exit from it just use `deactivate`.

**Reply by joshikida** [reply]
Oh man I feel stupid. I wasn't running the interpreter I downloaded discord.py to. Thank you so much!
